src/solvers/dgmulti/dg.jl

```diff
@@ -0,0 +1,314 @@
+# !!! warning "Experimental features"
+
+# out <- A*x
+mul_by!(A) = let A = A
+  @inline (out, x)->matmul!(out, A, x)
+end
+
+# specialize for SBP operators since `matmul!` doesn't work for `UniformScaling` types.
+mul_by!(A::UniformScaling) = let A = A
+  @inline (out, x)->mul!(out, A, x)
+end
+
+# # Todo: simplices. Use `matmul!` for the following 2 functions until 5-arg `matmul!` once
+# the hanging bug is fixed (see https://github.com/JuliaLinearAlgebra/Octavian.jl/issues/103).
+
+# out <- out + A * x
+mul_by_accum!(A) = let A = A
+  @inline (out, x)->mul!(out, A, x, one(eltype(out)), one(eltype(out)))
+end
+
+#  out <- out + α * A * x
+mul_by_accum!(A, α) = let A = A
+  @inline (out, x)->mul!(out, A, x, α, one(eltype(out)))
+end
```

Comment:
I'm pretty sure the `let` blocks don't do anything here - they are inside the function where a new scope is already introduced. Could you please check that and remove the `let` blocks if you don't observe performance regressions?

Reply:
You're right, seems like the `let` blocks aren't needed. Thanks!